static/js/panels.js

```diff
@@ -84,6 +86,18 @@ exports.initialize = function () {
         resize_app();
     });
 
+    $(".accept-bankruptcy").on("click", function (e) {
+        e.preventDefault();
+        $(this).closest(".alert").hide();
+        $('.bankruptcy-loader').show();
+        setTimeout(pointer.fast_forward_pointer, 1000);
+        resize_app();
+    });
+
+    $("#bankruptcy").on("hide", function () {
+        exports.enable();
```

Comment:
This should be `unread_ui.enable` -- when you move code from one module to another, `exports` refers to the current module and thus stops meaning the same thing.